### ISSUE TITLE
signal strength

### ISSUE THREAD

**Issue by darkalexpp** [aircrack-ng/rtl8188eus#46]
Can the antenna power be increased from the driver?

**Comment by irabz** [comment]
No, but you can mount a different antenna with higher range on the adapter (ex: one from an Alfa, or even one from a Router works)

**Comment by SisyphusIsntHappy** [comment]
Why, is the chipset incapable of increasing the power or is it just the driver's fault?
